openquake/commonlib/readinput.py

```diff
@@ -646,7 +646,8 @@ def get_exposure(oqparam):
     """
     out_of_region = 0
     if oqparam.region_constraint:
-        region = wkt.loads(oqparam.region_constraint)
+        region = wkt.loads(oqparam.region_constraint).buffer(0.1)
+        # there is a small buffer to accept points on the fence
```

Comment:
This might include points that are very close to the fence. Please add something further to this comment.